### PR TITLE
TF-196 CTI files are accessible from Settings UI

### PR DESCRIPTION
**Pull request by MattGrayUL**
Moved CTI files to Settings UI app

- [x] Test plan updated (by PR creator)
- [x] Code Review
- [x] Developer Testing: Key item tests
- [x] Developer Testing: Execution of the updated test plan (changed items for this work only)
- [ ] Product Owner Review (Optional)

### CODE REVIEW

File: TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceShared/Scripts/Configuration/Managers/TFAppConfig.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public class ConfigurationState
 
         // CTI Settings
         public bool ctiEnabled = false;
-        public string ctiFilePath = Path.Combine(Application.streamingAssetsPath, "CallToInteract", "AirPush_Portrait.mp4");
+        public string ctiFilePath = Path.Combine(Application.streamingAssetsPath, "CallToInteract/AirPush_Portrait.mp4");
```
Comment (PeterNancollis-UH) [inline TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceShared/Scripts/Configuration/Managers/TFAppConfig.cs:55]: Why is this change necessary? Isn't the result the same either way?

Comment (MattGrayUL) [reply TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceShared/Scripts/Configuration/Managers/TFAppConfig.cs:55]: Well, both work as paths but it removes one of the double slashes we get in the config file, for some reason Path.Combine likes to add extra slashes

**Review by PeterNancollis-UH — APPROVED** [review]
